test/Timer.spec.js

```diff
@@ -0,0 +1,23 @@
+import { Timer, countdown } from '../src/app/components/Timer';
+import '@testing-library/jest-dom';
+import { screen } from '@testing-library/dom';
+
+describe('Timer component', () => {
+  beforeEach(jest.useFakeTimers);
```

Comment:
Aaa nie zwrocilem uwagi na ten element - to trzeba to przedyskutowac.
Ale raczej timer powinien do naszego TextTimer jakos przychodzic z zewnatrz - bo musimy miec to samo zrodlo dla textu i lightsabera.